### PR TITLE
Gold standard import

### PR DESCRIPTION
**Pull request by brian-c**
Adds a simple importer for gold standard classifications, and a toggle for optional classification feedback.

Please review after zooniverse/Panoptes#1524 is merged for the toggle to work.

### CODE REVIEW

**Comment by marten** [comment]
Panoptes-side is merged

File: app/pages/lab/gold-standard-importer.cjsx [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+    catch parseError
+      @state.errorsByFile[file.name] = parseError
+      @setState errorsByFile: @state.errorsByFile
+
+  startImport: ->
+    @setState
+      progress: {}
+    Object.keys(@state.classificationsByFile).forEach (fileName) =>
+      classifications = @state.classificationsByFile[fileName]
+      [Promise.resolve(), classifications...].reduce (awaitPreviousSave, classification, i1) =>
```
Comment (aweiksnar) [inline app/pages/lab/gold-standard-importer.cjsx]: If you supply the accumulator as a third arg to reduce here the index should be correct

File: app/pages/lab/gold-standard-importer.cjsx [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+
+module.exports = React.createClass
+  IN_PROGRESS: 'IN_PROGRESS'
+  SUCCESSFUL: 'SUCCESSFUL'
+
+  getDefaultProps: ->
+    project: null
+    workflow: null
+    files: []
+    onClose: ->
```
Comment (aweiksnar) [inline app/pages/lab/gold-standard-importer.cjsx]: Nice syntax for this, hadn't thought of that before

**Comment by aweiksnar** [comment]
Works well!! It may be worth messaging that the data should be in a `.json` file in that `?` box too just in case people aren't sure

**Comment by brian-c** [comment]
@aweiksnar I swapped that `reduce` syntax and rebased, thanks.

**Comment by aweiksnar** [comment]
Looks good! thanks :shipit:

**Pull request merged**